code/game/objects/items/weapons/his_grace.dm

```diff
@@ -20,14 +20,14 @@
 	var/prev_bloodthirst = HIS_GRACE_SATIATED
 	var/force_bonus = 0
 
-/obj/item/weapon/his_grace/New()
+/obj/item/weapon/his_grace/Initialize()
 	..()
 	START_PROCESSING(SSprocessing, src)
 
 /obj/item/weapon/his_grace/Destroy()
 	STOP_PROCESSING(SSprocessing, src)
 	for(var/mob/living/L in src)
-		L.forceMove(get_turf(src))
+		L.forceMove(loc)
```

Comment:
You're right, but we need a better way to decide when to forceMove something to a loc or get_turf